.gitignore

```diff
@@ -19,3 +19,7 @@ coverage
 
 # bundled files
 lib/bundle.js
+
+package-lock.json
+
+front-end-testing/test/*
```

Comment:
The solution still exists in the `test-complete` directory and they are pointed towards it in the README in case they get stuck :)